test/lit.cfg

```diff
@@ -592,12 +592,13 @@ else:
     if kIsWindows:
         config.swift_driver += " -libc " + config.swift_stdlib_msvc_runtime
     config.swiftc_driver = (
-        "%r %s %s %s %s" % (
+        "%r %s %s %s %s %s" % (
             config.swiftc,
             '' if kIsWindows else '-toolchain-stdlib-rpath',
             mcp_opt,
             config.swift_test_options,
             config.swift_driver_test_options,
+            config.swift_system_overlay_opt,
```

Comment:
this is empty in non-Windows platforms, defined at:
https://github.com/apple/swift/blob/3fe8b713d476dcbcf700588acf14f120ea5bfd21/test/lit.cfg#L379-L389